github-pr-reviewer/code_reviewer/github_code.py

```diff
@@ -0,0 +1,28 @@
+from github import Github
+
+from code_reviewer.reviewer import CodeReviewer
+
+class GithubAPI:
+    def __init__(self, access_token):
+        self.g = Github(access_token)
+        self.code_reviewer = CodeReviewer()
+    
+    def write_comments_for_pr(self, repo_name, pr_number):
+        repo = self.g.get_repo(repo_name)
```

Comment:
Add a docstring to the write_comments_for_pr method to describe its purpose, the parameters, and the return value.